.github/workflows/js-ci.yml

```diff
@@ -233,6 +233,7 @@ jobs:
         env:
           KEYCLOAK_ADMIN: admin
           KEYCLOAK_ADMIN_PASSWORD: admin
+          JAVA_OPTS_APPEND: -Djava.net.preferIPv4Stack=true
```

Comment:
@jonkoops Do you see any issues after #21340 was merged?

Reply:
Yes, the Cypress action we're using is unable to verify the Keycloak server is running and times out. You can see it happening in [some of our runs](https://github.com/keycloak/keycloak/actions/runs/5458794350/jobs/9934258722?pr=21405). Not sure whether this is the cause, but it's a likely suspect.

Reply:
Upon closer inspection I am getting the following error:
```log
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/keycloak/quarkus/runtime/KeycloakMain has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
	at io.quarkus.bootstrap.runner.RunnerClassLoader.loadClass(RunnerClassLoader.java:105)
	at io.quarkus.bootstrap.runner.RunnerClassLoader.loadClass(RunnerClassLoader.java:65)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:60)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
```

Reply:
If Cypress has issues, maybe users will too?

Reply:
Looks to be that the Java version is mismatched in different parts of the workflow, so this should not affect users.